### PR TITLE
Add Arcadian GT Optional

### PR DESCRIPTION
**Pull request by TyroneMF**
- Reworked sounds
- Removed sidepanels
- Reworked speeds
- Simulated runflat tires

### CODE REVIEW

**Comment by TyroneMF** [comment]
Last commit should have what you requested.

File: optionals/arcadian/CfgVehicles.hpp [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+            class HitPoints;
+        };
+    };
+    class Eventhandlers;
+    class AnimationSources;
+};
+
+class CUP_SUV_Unarmed_Base: CUP_SUV_Base {
+    class AnimationSources: AnimationSources {
+        class rear_window_source {
```
Comment (jonpas) [inline optionals/arcadian/CfgVehicles.hpp]: Why is this required?

File: optionals/arcadian/CfgVehicles.hpp [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+    };
+};
+
+class CLASS(SUV_Base): CUP_SUV_Unarmed_Base {
+    armor = 90;
+    crewCrashProtection = 0.25;
+    displayName = "GT SUV Base";
+    hiddenSelections[] = { "camo" };
+    HiddenSelectionsTextures[] = { "\x\tacs\addons\arcadian\data\arcadian_body_black_co.paa" };
+    // Damage & Hitpoints
```
Comment (jonpas) [inline optionals/arcadian/CfgVehicles.hpp]:
```suggestion

    // Damage & Hitpoints
```

File: optionals/arcadian/CfgVehicles.hpp [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+            animPeriod = 1;
+        };
+    };
+};
+
+class CLASS(SUV_Base): CUP_SUV_Unarmed_Base {
+    armor = 90;
+    crewCrashProtection = 0.25;
+    displayName = "GT SUV Base";
+    hiddenSelections[] = { "camo" };
```
Comment (jonpas) [inline optionals/arcadian/CfgVehicles.hpp]: Get rid of spaces in array definitions, everywhere.
```suggestion
    hiddenSelections[] = {"camo"};
```
Is this hidden selection not already defined in `CUP_SUV_Unarmed_Base`.

File: optionals/arcadian/CfgVehicles.hpp [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+    };
+    wheelDamageThreshold = 0.18000001;
+    wheelDamageRadiusCoef = 0.75;
+    wheelDestroyRadiusCoef = 0.47999999;
+    soundEngineOnInt[] = { "A3\Sounds_F\vehicles2\soft\Mrap_01\Mrap_01_Engine_Int_Start",0.630957,1 };
+    soundEngineOnExt[] = { "A3\Sounds_F\vehicles2\soft\Mrap_01\Mrap_01_Engine_Ext_Start",1.99526,1,50 };
+    soundEngineOffInt[] = { "A3\Sounds_F\vehicles2\soft\Mrap_01\Mrap_01_Engine_Int_stop",0.501187,1 };
+    soundEngineOffExt[] = { "A3\Sounds_F\vehicles2\soft\Mrap_01\Mrap_01_Engine_Ext_stop",1.99526,1,50 };
+
+#include "sounds.hpp"
```
Comment (jonpas) [inline optionals/arcadian/CfgVehicles.hpp]:
```suggestion
    #include "sounds.hpp"
```

File: optionals/arcadian/CfgVehicles.hpp [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+
+    thrustDelay = 0.1;
+    brakeIdleSpeed = 1.78;
+    maxSpeed = 200;
+    fuelCapacity = 45;
+    idleRpm = 800;
+    redRpm = 4500;
+    // Source, Vanilla Hunter
```
Comment (jonpas) [inline optionals/arcadian/CfgVehicles.hpp]:
```suggestion

    // Source: Vanilla Hunter
```

File: optionals/arcadian/CfgVehicles.hpp [hunk 6]
```diff
@@ -0,0 +1,235 @@
...
+    maxOmega = 450;
+    peakTorque = 425;
+    dampingRateFullThrottle = 0.079999998;
+    dampingRateZeroThrottleClutchEngaged = 0.5;
+    dampingRateZeroThrottleClutchDisengaged = 0.34999999;
+    torqueCurve[] = { { "(0/3500)","(0/425)" }, { "(500/3500)","(200/425)" }, { "(1500/3500)","(405/425)" }, { "(2000/3500)","(425/425)" }, { "(2500/3500)","(350/425)" }, { "(3000/3500)","(300/425)" }, { "(6000/3500)","(0/425)" } };
+    changeGearMinEffectivity[] = {0.94999999,0.15000001,0.94999999,0.94999999,0.94999999,0.94999999,0.94999999}; // Default {1,0.15000001,1,1,1,1,1,1};
+    switchTime =  0.31;
+    latency =  1.5;
+    ace_refuel_fuelCapacity = 60;
```
Comment (jonpas) [inline optionals/arcadian/CfgVehicles.hpp]: Do we change that?

File: optionals/arcadian/CfgVehicles.hpp [hunk 7]
```diff
@@ -0,0 +1,235 @@
...
+
+    class CargoTurret;
+    class Turrets: Turrets {
+    delete CargoTurret_03;
```
Comment (jonpas) [inline optionals/arcadian/CfgVehicles.hpp]: Indent

File: optionals/arcadian/CfgVehicles.hpp [hunk 8]
```diff
@@ -0,0 +1,235 @@
...
+    delete CargoTurret_04;
+    delete CargoTurret_05;
+    delete CargoTurret_06;
+    delete CargoTurret_07;
+    delete CargoTurret_08;
+        class CargoTurret_09: CargoTurret {
+        minElev = -10;
```
Comment (jonpas) [inline optionals/arcadian/CfgVehicles.hpp]: Indent

File: optionals/arcadian/CfgVehicles.hpp [hunk 9]
```diff
@@ -0,0 +1,235 @@
...
+
+    class CargoTurret;
+    class Turrets: Turrets {
+    delete CargoTurret_03;
+    delete CargoTurret_04;
+    delete CargoTurret_05;
+    delete CargoTurret_06;
+    delete CargoTurret_07;
+    delete CargoTurret_08;
+        class CargoTurret_09: CargoTurret {
```
Comment (jonpas) [inline optionals/arcadian/CfgVehicles.hpp]: Do we change anything here at all?

File: optionals/arcadian/config.cpp [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+            QCLASS(Arcadian_B_SUV),
+            QCLASS(Arcadian_B_SUV_Tan),
+            QCLASS(Arcadian_B_SUV_Green),
+            QCLASS(Arcadian_I_SUV_Black),
+            QCLASS(Arcadian_I_SUV_Tan),
+            QCLASS(Arcadian_I_SUV_Green)
+        };
+        weapons[] = { };
+        requiredVersion = 0.1;
+        requiredAddons[] = { "tacs_main", "CUP_WheeledVehicles_Core", "CUP_Wheeled_SUV" };
```
Comment (jonpas) [inline optionals/arcadian/config.cpp]: `CUP_Wheeled_SUV` most likely already requires `CUP_WheeledVehicles_Core`.

File: optionals/arcadian/config.cpp [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+        author = ECSTRING(main,Author);
+        authors[] = {"TyroneMF", "GilleeDoo"};
+        url = ECSTRING(main,URL);
+        VERSION_CONFIG;
+    };
+};
+
+#include "CfgVehicles.hpp"
```
Comment (jonpas) [inline optionals/arcadian/config.cpp]:
```suggestion
#include "CfgVehicles.hpp"

```

File: optionals/arcadian/script_component.hpp [hunk 12]
```diff
@@ -0,0 +1,6 @@
+#define COMPONENT gt armory arcadian
```
Comment (jonpas) [inline optionals/arcadian/script_component.hpp]: This can't have spaces.
```suggestion
#define COMPONENT arcadian
```

File: optionals/arcadian/script_component.hpp [hunk 13]
```diff
@@ -0,0 +1,6 @@
...
+#define COMPONENT_BEAUTIFIED GT Armory Arcadian
```
Comment (jonpas) [inline optionals/arcadian/script_component.hpp]:
```suggestion
#define COMPONENT_BEAUTIFIED GT Arcadian
```

File: optionals/arcadian/CfgVehicles.hpp (hunk 1 above)
Comment (TyroneMF) [reply optionals/arcadian/CfgVehicles.hpp]: Because the rear window doesn't work without it.

File: optionals/arcadian/CfgVehicles.hpp (hunk 9 above)
Comment (TyroneMF) [reply optionals/arcadian/CfgVehicles.hpp]: I named the rear seat. But after removing everything but the seat name it stopped working.

File: optionals/arcadian/CfgVehicles.hpp (hunk 6 above)
Comment (TyroneMF) [reply optionals/arcadian/CfgVehicles.hpp]: Yes
